packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -32,32 +39,37 @@ export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBox
           }
         },
       hasInput && {
-        borderColor: palette.neutralSecondary
+        borderColor: inputBorderHasInput
       },
       hasFocus && {
-        borderColor: palette.themePrimary
+        borderColor: semanticColors.inputFocusBorderAlt
       },
       underlined && {
         borderRadius: 0
       }
     ],
-    iconContainer: {
-      color: palette.themeDarkAlt
-    },
+    iconContainer: [
```

Comment:
This rule can be simplified to make it much more readable
```tsx
  iconContainer: {
    color: disabled ? inputIconDisabled : inputIcon
  }
```

Reply:
Nice! I'll pop the change in, thanks @Vitalius1 .